src/ShopBundle/objects/ArticleList/DatabaseAccessLayer/SortType.php

```diff
@@ -32,4 +42,32 @@ public function getSortString()
 
         return implode(', ', $parts);
     }
+
+    private function getMultilingualSortString(string $sortString): string
```

Comment:
There is an easier way for this. Use the FieldTranslationUtil. Either getTranslatedQuery() or getTranslatedFieldName

Reply:
Used the getTranslatedFieldName function like you suggested